Modules/CalcOffence-3_0.lua

```diff
@@ -2225,8 +2236,12 @@ function calcs.offence(env, actor, activeSkill)
             local storedDamageMore = round(skillModList:More(cfg, "ImpaleEffect"), 2)
             local storedDamageModifier = (1 + storedDamageInc) * storedDamageMore
             local impaleStoredDamage = baseStoredDamage * storedDamageModifier
+            local impaleHitDamageMod = impaleStoredDamage * impaleStacks  -- Source: https://www.reddit.com/r/pathofexile/comments/chgqqt/impale_and_armor_interaction/
+
+            local impaleArmourReduction = calcs.armourReductionF(round(calcLib.val(enemyDB, "Armour")), impaleHitDamageMod * output.impaleStoredHitAvg)
+            local impaleResist = m_max(0, enemyDB:Sum("BASE", nil, "PhysicalDamageReduction") + skillModList:Sum("BASE", cfg, "EnemyPhysicalDamageReduction") + skillModList:Sum("BASE", cfg, "EnemyImpalePhysicalDamageReduction") + impaleArmourReduction)
```

Comment:
The Physical Damage Reduction for Impales should be shown in the Impale multiplier breakdown. Can add something like this to line 2263 `t_insert(breakdown.ImpaleModifier, s_format("x %.2f ^8(impale physical damage reduction)", (1 - impaleResist / 100)))`

Reply:
"impale enemy physical damage reduction"...too wordy?